Bio/PDB/internal_coords.py

```diff
@@ -1933,23 +1932,23 @@ def pick_angle(
         The following are equivalent (except for sidechains with non-carbon
         atoms for chi2):
```

Comment:
This probably needs a double colon if you want the next block to be an RST literal block.

Reply:
Thanks, fixed and I knew I needed to check through the html for the API docs :-)

What's bothering me is that I'm not seeing the same flake8 issues:

    rob@water:03 19:06:51 biopython $ flake8 --version
    3.7.9 (mccabe: 0.6.1, pycodestyle: 2.5.0, pyflakes: 2.1.1) CPython 3.7.6 on Darwin
    rob@water:03 19:07:00 biopython $ flake8 Bio/PDB/internal_coords.py 
    rob@water:03 19:07:05 biopython $ 

I went through the list for the travis output of 'style installdeps' and ensured I had them all.

Reply:
You are missing ``flake8-rst-docstrings`` which gives the ``RST`` prefix codes (the point of this one is to catch things which will upset Sphinx and the HTML API docs), and a bunch more, see:

https://github.com/biopython/biopython/blob/master/CONTRIBUTING.rst